### ISSUE TITLE
Add datahub into the chart

### ISSUE THREAD

**Issue by edevosc2c** [georchestra/helm-georchestra#34]
The official docker projet for geOrchestra has datahub in its docker-compose: https://github.com/georchestra/docker/blob/master/docker-compose.yml#L229

If datahub is becoming a component of the geOrchestra stack, then we maybe should implement it into the helm chart?

We could have a [dependency](https://helm.sh/docs/helm/helm_dependency/) on the official datahub helm chart when it is out: https://github.com/orgs/geonetwork/discussions/7097

ping @fgravin @pmauduit @jeanmi151

**Comment by fvanderbiest** [comment]
I'd rather have it as a dependency, because the datahub will be deployed outside geOrchestra too.

**Comment by fgravin** [comment]
Yes, datahub should have its helm chart in the geonetwork organization repository. Hopefully, it could be used in Geocat directily as it is ?
Then, it would need some adaptation for the georchestra version maybe, because of the way the `config.toml` file is fetched.

**Comment by fgravin** [comment]
The helm chart repo has been published in the GeoNetwork organization under https://github.com/geonetwork/geonetwork-helm-charts/tree/main/datahub

I think we should consider that the Datahub is part of geOrchestra, and move forward with its integration within the offical geOrchestra helm charts :+1:

**Comment by jeanpommier** [comment]
+1

On a standard georchestra deployment with this chart, datahub is the default home page. But not installed with it. 
Not a big deal, but not very consistent either
